WcaOnRails/app/webpacker/components/Panel/Leader/GroupsManager.jsx

```diff
@@ -44,6 +44,7 @@ function GroupTable({ groupId }) {
 export default function GroupsManager({ loggedInUserId }) {
   const { data: roles, loading, error } = useLoadedData(apiV0Urls.userRoles.listOfUser(
     loggedInUserId,
+    null, // Sort params
     { isActive: true, isGroupHidden: false, status: 'leader' },
   ));
```

Comment:
Then I would recommend either creating a new parameter object so that `{ sort: null }` can be passed in, or extracting the sort as a constant each time so that the const name replaces the need for a comment:
```suggestion
  const noSort = null;
  const { data: roles, loading, error } = useLoadedData(apiV0Urls.userRoles.listOfUser(
    loggedInUserId,
    noSort,
    { isActive: true, isGroupHidden: false, status: 'leader' },
  ));
```

Reply:
Okay, in that case I think I'll have a sort here as well so that the list of groups come in alphabetical order. I've added sort now. :-)